src/components/partner/dto/list-partner.dto.ts

```diff
@@ -0,0 +1,37 @@
+import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import {
+  PaginatedList,
+  SecuredList,
+  SortablePaginationInput,
+} from '../../../common';
+import { Partner } from './partner';
+
+@InputType()
+export abstract class PartnerFilters {
+  @Field({ nullable: true })
+  readonly organizationId?: string;
```

Comment:
We aren't exposing ID filters through the API now. 

If you need them for internal use just remove the field decorators.
```suggestion
  readonly organizationId?: string;
```